db/migrate/20181127012722_add_guid_to_notifications.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddGuidToNotifications < ActiveRecord::Migration[5.1]
+  def change
+    add_column :notifications, :guid, :string
```

Comment:
@frankrousseau I'm doing it in the approved way so they should be okay with it.  It is required to implement the API because everything besides the Aspects are supposed to work through GUIDs.